.gitignore

```diff
@@ -7,6 +7,5 @@ coverage/
 # See https://docs.npmjs.com/misc/scripts
 client/fonts/
 client/js/bundle.js
-client/js/libs.min.js.map
```

Comment:
This webpack config doesn't produce source maps, but no reason it couldn't--I'll update it to produce inline source maps (so no need for an extra file).